build/Dockerfile

```diff
@@ -39,6 +40,7 @@ COPY utils/scripts/logger.py  utils/scripts/logger.py
 COPY utils/scripts/util.py  utils/scripts/util.py
 RUN  chmod +x run.sh
 COPY requirements.txt setup.py .
+RUN  python3 --version
```

Comment:
Do you need to do this if you don't check the version? Is this just to have it logged somewhere?

Reply:
it's not needed, but I wanted to have this logged. Do you prefer to remove it?

Reply:
No, having it logged might be useful if we ever run into weird compatibility issues